crates/bevy_pbr/src/entity.rs

```diff
@@ -26,21 +26,7 @@ impl Default for PbrComponents {
         Self {
             render_pipelines: RenderPipelines::from_pipelines(vec![RenderPipeline::specialized(
```

Comment:
We can replace the `specialized()` constructor with `new()` now.

Reply:
done